### PR TITLE
Component/person card

### PR DESCRIPTION
**Pull request by suejinkim20**
- personCard component created
- job titles rendered conditionally based on showTitle prop, can easily be changed by altering the boolean
- personCard component incorporated to view OOD photos on people page (with job titles)
- personCard component incorporated to view all staff photos on people page (with job titles)
- personCard component incorporated on all research groups pages (no job titles)
- personCard component incorporated on all operations teams pages (with job titles)

### CODE REVIEW

File: components/person-card.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { Box, Card, CardHeader, CardMedia, CardContent, Grid, Typography } from '@mui/material'
+
+import { Link, Page } from './link'
```
Comment (mbwatson) [inline components/person-card.js]: what's this Page import?

**Review by mbwatson — CHANGES_REQUESTED** [review]
this looks perfect! i think there's just one code thing i had a question about that i commented on specifically.

File: components/person-card.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <Typography>{ person.title }</Typography>
+          </CardContent>
+        </Card>
+    </Fragment>
+  )
+}
+
+PersonCard.propTypes = {
+  person: PropTypes.shape({
+    slug: PropTypes.string,
```
Comment (mbwatson) [inline components/person-card.js]: all of these should be required props i believe, right?

Comment (suejinkim20) [reply components/person-card.js]: ✅

File: components/person-card.js (hunk 1 above)
Comment (suejinkim20) [reply components/person-card.js]: removed ✅

**Comment by mbwatson** [comment]
i love it. let's go ahead and get the generic avatar image in this PR (or just grey placeholder div will be fine). you can pick.

File: components/person-card.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  )
+}
+
+PersonCard.propTypes = {
+  person: PropTypes.shape({
+    slug: PropTypes.string.isRequired,
+    firstName: PropTypes.string.isRequired,
+    lastName: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    photo: PropTypes.object.isRequired,
```
Comment (mbwatson) [inline components/person-card.js]: those without photos throw an error now since photo is required. maybe photo is the only non-required one

**Review by mbwatson — CHANGES_REQUESTED** [review]

File: components/person-card.js (hunk 3 above)
Comment (suejinkim20) [reply components/person-card.js]: ✅

File: components/person-card.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+PersonCard.propTypes = {
+  person: PropTypes.shape({
+    slug: PropTypes.string.isRequired,
+    firstName: PropTypes.string.isRequired,
+    lastName: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    photo: PropTypes.object,
+  }),
+}
```
Comment (mbwatson) [inline components/person-card.js]: we should add a prop-type declaration for showTitle. is it required? should we define a default value?

**Review by mbwatson — APPROVED** [review]
@suejinkim20 this is great! I'm not sure about the flexy grid on the people's photos yet (which might just feel a bit off because all the photos are different sizes and aspect ratios), but that's just a small detail we can explore down the road. :+1:

**Pull request merged**